### PR TITLE
[MDCT-2923] Section H with new part 2

### PR DESCRIPTION
**Pull request by BearHanded**
### Description

- Adds new Part 2
- Updates existing part 2 -> part 3
- Updates text overall

### Related ticket(s)

MDCT-2923

---
### How to test

- Compare the diffs
- pull it down
- release 2022 and 2023
- Try section 1 q 1 Yes and no, check for conditional rendering
- Compare against request

### Important updates

---
### Author checklist

- [x] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/database/data/seed/seed-section-base-2023.json [hunk 1]
```diff
@@ -8411,7 +8411,7 @@
           },
           {
             "id": "2023-03-h",
-            "text": "Children’s Health Insurance Program Reauthorization Act (CHIPRA) requires that all CHIP programs submit survey results from the Consumer Assessment of Healthcare Providers and Systems (CAHPS). The survey assesses your CHIP program quality and customer satisfaction. For the 2022 CARTS report, the only option for reporting CAHPS results will be through the submission of raw data to the Agency for Healthcare Research and Quality (AHRQ) CAHPS Database.",
+            "text": "Children’s Health Insurance Program Reauthorization Act (CHIPRA) requires that all CHIP programs submit survey results from the Consumer Assessment of Healthcare Providers and Systems (CAHPS). The survey assesses your CHIP program quality and customer satisfaction. For the 2024 CARTS report, the only option for reporting CAHPS results will be through the submission of raw data to the Agency for Healthcare Research and Quality (AHRQ) CAHPS Database.",
```
Comment (gmrabian) [inline services/database/data/seed/seed-section-base-2023.json:8414]: what a block of text 😵

File: services/database/data/seed/seed-section-base-2023.json [hunk 2]
```diff
@@ -8473,12 +8474,141 @@
...
                 "id": "2023-03-h-02",
-                "text": "Since you didn’t collect the CAHPS survey, please complete Part 2.",
                 "type": "part",
-                "title": "You didn't collect the CAHPS survey",
+                "title": "You collected the CAHPS survey",
                 "questions": [
                   {
                     "id": "2023-03-h-02-01",
+                    "type": "file_upload",
+                    "label": "Upload a summary report of your CAHPS survey results. This is optional if you already submitted CAHPS raw data to the AHRQ CAHPS database. Submit results only for the CHIP population, not for both Medicaid (Title XIX) and CHIP (Title XXI) together. Your data should represent children enrolled in all types of delivery systems (Managed Care, PCCM, and Fee for Service).",
```
Comment (gmrabian) [inline services/database/data/seed/seed-section-base-2023.json:8483]: Fee-for-Service was hyphenated in [Section 1](https://qmacbis.atlassian.net/browse/MDCT-2910?atlOrigin=eyJpIjoiZjFiMzUyNzNiMWI4NDAyOWEwMGU2MDM5YzNiNjMyMzEiLCJwIjoiaiJ9) so just checking if we want that here

**Review by gmrabian — COMMENTED** [review]
There were updates to the upload box text in the ticket that I didn't see in here. Otherwise looks good.

**Review by gmrabian — DISMISSED** [review]

**Review by benmartin-coforma — COMMENTED** [review]
I haven't tested this, but I found a couple of nits to pick.

File: services/database/data/seed/seed-section-base-2023.json
[the remark this answers is not in the document]
Comment (BearHanded) [reply services/database/data/seed/seed-section-base-2023.json:1]: I can double check on this, but this should be fine as far as using out pre-built components goes.

**Review by benmartin-coforma — DISMISSED** [review]

File: services/database/data/seed/seed-section-base-2023.json
[the remark this answers is not in the document]
Comment (BearHanded) [reply services/database/data/seed/seed-section-base-2023.json:1]: Verified with David and he requested the new text, good catch

**Review by benmartin-coforma — APPROVED** [review]

**Review by gmrabian — APPROVED** [review]

**Pull request merged**